content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -1,8 +1,8 @@
-# Submitting to NMDC
+# Submitting to the NMDC
 
 
-## Introduction
-The NMDC has developed two products for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) app (currently in beta testing). Each product was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary metadata describing a study and its biosamples.
+## Introduction: Making metadata capture easy
+The NMDC has developed two integrated resources for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) mobile app (released April 2025). Each resource was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary information describing a study, its biosamples, and associated assays.
```

Comment:
"assays" feels very sequencing focused?
analyses? (Asked in google doc, might update)